resources/function_help/json/is_empty

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "is_empty",
+  "type": "function",
+  "description": "Returns true if a geometry is empty (without coordinates) or false if the geometry is empty.",
```

Comment:
Isn't there an issue in this sentence? same condition returns either true or false?

Reply:
Done. Thanks...